addons/message_survey_staging_10s/manifest.json

```diff
@@ -12,8 +12,21 @@
     "id": "message_survey_staging_10s",
     "title": "Your subscription is ending soon!",
     "subtitle": "Your Mozilla VPN subscription is ending soon. If you don’t want to say goodbye you can easily renew your subscription below.",
-    "date": 1641406997,
+    "date": 1661367590,
+    "badge": "critical",
     "blocks": [
+      { "id": "c_1",
+      "type": "button",
+      "style": "primary",
+      "content": "Renew subscription",
+      "javascript": "PresentSubscriptionManagementScreen.js"
+      },
+      { "id": "c_2",
+        "type": "button",
+        "style": "link",
+        "content": "Get help",
+        "javascript": "PresentGetHelpScreen.js"
```

Comment:
can you revert all of these changes? And file new messages for staging if you want to test things.
Asking this because this survey has been renamed and I guess you will have a few conflict to fix.
I also would like to make this add-on a survey one: label, text, etc.